### DISCUSSION TITLE
Deleting Classes in DB through python

### DISCUSSION THREAD

**Discussion by IngridJSJ** [Spine-project/Spine-Toolbox#1753] [Support]
Hello!
I am simulations in loop for many years. Every year the data increases. There are some classes that could be erased from the DB each year. 
Do you have an example of how to erase classes through a python script? 
Thank you very much!

**Comment by soininen** [comment]
This has been asked before in #1739. Does the example script there do what you want or do you have other requirements?

**Reply by IngridJSJ** [reply]
Aa sorry I overseen that answer. I was in holidays then :)
